.github/dependabot.yml

```diff
@@ -21,6 +21,8 @@ updates:
   - "area/dependency"
   - "kind/task"
   - "ok-to-test"
+  # 'do-not-merge/hold' because we also need to update the 'scripts/generate-mod.sh' file (and re-run it) accordingly
```

Comment:
Do we need this commented line or did you forget to remove it?

Reply:
Nope, I didn't forget to remove it ^^. It is intentional, so we understand why we need this label.
Ideally, a comment in the PR would be better but this is not possible to do automatically at this time in the configuration options.